static/search.html

```diff
@@ -105,6 +109,9 @@
             <th>
                 Serial
             </th>
+            <th>
+                Lineage
+            </th>
```

Comment:
Not sure if it ever makes sense to display the  Lineage  in the interface. For me, it's an internal ID that Terraform users don't care about.

Reply:
Ok so I remove it